### PR TITLE
Refer to RFC 2119, and fix a couple of typos

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: document/Normative.mdk [hunk 1]
```diff
@@ -6,20 +6,23 @@ For dated references, only the edition cited applies.
...
 
-JSON Schema: core definitions and terminology \[viewed 2016-04-22\]. Available from &lt;<http://json-schema.org/latest/json-schema-core.html>&gt;
+JSON Schema: core definitions and terminology \[viewed 2016-04-22\]. Available from <http://json-schema.org/latest/json-schema-core.html>
 
 RFC 2045, Multipurpose Internet Mail Extensions (MIME) Part One: Format of Internet Message Bodies.
-Available from &lt;<http://www.ietf.org/rfc/rfc2045.txt>&gt;
+Available from <http://www.ietf.org/rfc/rfc2045.txt>
+
+RFC 2119, Key words for use in RFCs to Indicate Requirement Levels.
+Available from <https://www.ietf.org/rfc/rfc2119.txt>
```
Comment (ghost) [inline document/Normative.mdk:23]: This

File: document/Conventions.mdk [hunk 2]
```diff
@@ -4,6 +4,11 @@
 
 The following conventions are used within this document.
 
+## Key words for requirements levels
+
+In this document, the key words "must", "must not", "required", "shall", "shall not",
+"should", "should not", "recommended", "may", and "optional" are used as defined in RFC 2119.
```
Comment (michaelcfanning) [inline document/Conventions.mdk:10]: >RFC 2119 [](start = 84, length = 8)

link to it? [https://www.ietf.org/rfc/rfc2119.txt](https://www.ietf.org/rfc/rfc2119.txt)

**Comment by michaelcfanning** [comment]
:shipit:

File: document/Conventions.mdk (hunk 2 above)
Comment (ghost) [reply document/Conventions.mdk:10]: There aren't any inline links from the spec's text to the references. All the links are in the Normative References section.

**Pull request merged**